recipes/pyflyby/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "pyflyby" %}
+{% set version = "1.6.2" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 5bebd9fe0f40d809bbf029e3d0d1bfa7bfe2a94379f5a8f383cb85d94fb9a88c
+
+build:
+  number: 0
+  noarch: python
+  entry_points:
+    - "py=pyflyby._py:py_main\npy3=pyflyby._py:py_main"
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - pytest
+    - python
+  run:
+    - pyflakes
+    - python
+
+test:
+  imports:
+    - pyflyby
+  commands:
+    - pip check
+    - py --help
+  requires:
+    - pip
+
+about:
+  home: https://pypi.org/project/pyflyby/
+  summary: pyflyby - Python development productivity tools, in particular automatic import management
+  license: MIT
+  license_file: doc/LICENSE.txt
+
+extra:
+  recipe-maintainers:
+    - asmeurer
+    - quarl
```

Comment:
@quarl is there anyone else who should be a maintainer of the conda-forge feedstock? The maintainership gives them push access to the feedstock repo, so that they can merge PRs updating the pyflyby version or fixing any issues with the conda package.

Reply:
@quarl needs to give the explicit OK here to be a maintainer

Reply:
Or we can remove right now and readd in the feedstock

Reply:
Maybe remove @quarl for now, so we can get this merged